### PR TITLE
update doc on RAID block size

### PR DESCRIPTION
**Pull request by wfeldt**
## Task

- https://trello.com/c/eixP9hzP

Update doc on RAID block size.

### CODE REVIEW

File: storage/Devices/MdImpl.cc [hunk 1]
```diff
@@ -805,6 +805,8 @@ namespace storage
 	// big can lead to severe problems later on, e.g. a partition not
 	// fitting anymore, we make a conservative calculation.
 
+        // For combining disks with different block sizes, see doc/md-raid.md.
```
Comment (ancorgs) [inline storage/Devices/MdImpl.cc]: Mixed indentation strikes back

File: doc/md-raid.md [hunk 2]
```diff
@@ -88,3 +88,18 @@ MdMembers can be used as generic Mds.
...
+
+This seems to work (at least most of the time) even if some parts of the
+RAID are not aligned to this block size.
+
+Md::add_device() takes care to update Md::Region to use the correct block
+size. It is, however not possible to combine regions with different block
+sizes (in libstorage-ng) if the start or length of a region are misaligned.
+
+This is, however, not really a problem here as the region start is always 0
+and the RAID size is an approximation anyway - so we might simply round down.
```
Comment (ancorgs) [inline doc/md-raid.md]: Rounding down sounds reasonable to me

Comment (ancorgs) [reply doc/md-raid.md]: Could we stop raising the `InvalidBlockSize` If we implement that rounding down?

Comment (wfeldt) [reply doc/md-raid.md]: Yes. But I'd like not to mix everything up. This is only the doc.Then change the code. Then the backport.

**Pull request merged**